.gitignore

```diff
@@ -0,0 +1,2 @@
+anishot.egg-info/
+build/
```

Comment:
Could you add __pycache__ here?

Reply:
Hi @sergey-surkov!

Sure, I've added some others usefull gitignore rules like, file swap from editors, tox unit tests temporary path (not used for the moment, but alrealdy ready for the futur).